File: SecuJournal/sjImage.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace SecuJournal
{
    public partial class sjImage : Form
    {
        public sjImage()
        {
            InitializeComponent();
        }

        private void RatioButton169_CheckedChanged(object sender, EventArgs e)
        {
            if (RatioButton169.Checked == true)
            {
                numericUpDownX.Enabled = false;
                numericUpDownY.Enabled = false;
            }
        }

        private void RatioButton43_CheckedChanged(object sender, EventArgs e)
        {
            if (RatioButton43.Checked == true)
            {
                numericUpDownX.Enabled = false;
                numericUpDownY.Enabled = false;
            }
        }

        private void RatioButtonCustom_CheckedChanged(object sender, EventArgs e)
        {
            if (RatioButtonCustom.Checked == true)
            {
                numericUpDownX.Enabled = true;
                numericUpDownY.Enabled = true;
            }
        }

        private void OKButton_Click(object sender, EventArgs e)
        {
            if (RatioButton169.Checked == true)
            {
                Properties.Settings.Default.SetImgWidth = 208;
                Properties.Settings.Default.SetImgHeight = 117;
            }
            else if (RatioButton43.Checked == true)
            {
                Properties.Settings.Default.SetImgWidth = 200;
                Properties.Settings.Default.SetImgHeight = 150;
            }
            else
            {
                Properties.Settings.Default.SetImgWidth = Convert.ToInt32(numericUpDownX.Value);
                Properties.Settings.Default.SetImgHeight = Convert.ToInt32(numericUpDownY.Value);
            }
        }
    }
}

File: SecuJournal/SJImage.vb
Public Class SJImage
    Private Sub RatioButton169_CheckedChanged(sender As Object, e As EventArgs) Handles RatioButton169.CheckedChanged
        If RatioButton169.Checked = True Then
            NumericUpDown1.Enabled = False
            NumericUpDown2.Enabled = False
        End If
    End Sub

    Private Sub OKButton_Click(sender As Object, e As EventArgs) Handles OKButton.Click
        If RatioButton169.Checked = True Then
            My.Settings.SetImgWidth = 208
            My.Settings.SetImgHeight = 117
        ElseIf RatioButton43.Checked = True Then
            My.Settings.SetImgWidth = 200
            My.Settings.SetImgHeight = 150
        Else
            My.Settings.SetImgWidth = NumericUpDown1.Value
            My.Settings.SetImgHeight = NumericUpDown2.Value
        End If
        Me.Close()
    End Sub

    Private Sub RatioButton43_CheckedChanged(sender As Object, e As EventArgs) Handles RatioButton43.CheckedChanged
        If RatioButton43.Checked = True Then
            NumericUpDown1.Enabled = False
            NumericUpDown2.Enabled = False
        End If
    End Sub

    Private Sub RatioButtonCustom_CheckedChanged(sender As Object, e As EventArgs) Handles RatioButtonCustom.CheckedChanged
        NumericUpDown1.Enabled = True
        NumericUpDown2.Enabled = True
    End Sub
End Class